doc/playbook.rst

```diff
@@ -65,6 +65,29 @@ Any subsequent builds of the service container will build against both bases.
 
 This is the only situation in which a change to ``BASE_TAG_LIST`` should be committed to ``main``.
 
+Release Procedure
+-----------------
+Releases are largely automated through GitHub Actions (see the `ci-release.yaml <https://github.com/lsst-dm/prompt_processing/actions/workflows/ci-release.yaml>`  workflow file for details). When a semantic version tag is pushed to GitHub, Prompt Processing Docker images are published on GitHub and Docker Hub with that version.
+
+Regular releases happen from the ``main`` branch after changes have been merged.
+From the ``main`` branch you can release a new major version (``X.0.0``), a new minor version of the current major version (``X.Y.0``), or a new patch of the current major-minor version (``X.Y.Z``).  Release tags are semantic version identifiers following the :pep:`440` specification.
```

Comment:
For the purpose of distinguishing between major and minor versions, does compatibility mean:

For Prompt Processing,
* ability to read fanned-out `nextVisit` messages
* schema compatibility of APDB and Alert Stream outputs
* no particular requirements on Butler outputs

For fan-out,
* ability to read incoming Summit messages
* schema compatibility of fanned-out messages

The question of compatibility for fanned-out messages is a bit problematic because, so long as we're just dumping them into a Python dataclass, almost any change is incompatible: removing a field will give an error unless we include a default, as usual for schemas, but _adding_ a field will give an error unconditionally.

Reply:
Fixed link to pep.  Also updated instructions to use `X.Y.Z` for releases without prepending `v` to be consistent with other release numbering schemes used by LSST.